vermouth/data/mappings/martini30/modifications.mapping

```diff
@@ -0,0 +1,150 @@
+[ modification ]
+[ from ]
+universal
+[ to ]
+martini30b32
+
+[ from blocks ]
+C-ter
+[ to blocks ]
+C-ter
+
+[ from nodes ]
+N
+HN
+
+[ from edges ]
+HN N
+N CA
+
+[ mapping ]
+CA BB
+C  BB
+O  BB
+OXT BB
+
+
+[modification]
+[ from ]
+universal
+[ to ]
+martini30b32
+[ from blocks ]
+N-ter
+[ to blocks ]
+N-ter
+
+[ from nodes ]
+C
+O
+
+[ from edges ]
+C O
+CA C
+
+[ mapping ]
+HN1 BB
+HN2 BB
+HN3 BB
+N   BB
+CA  BB
+C   BB
+O   BB
+
+;#################
+;# Martini 30dev #
+;#################
+
+[ modification ]
+[ from ]
+universal
+[ to ]
+martini30dev
+[ from blocks ]
+C-ter
+[ to blocks ]
+C-ter
+[ from nodes ]
+N
+HN
+[ from edges ]
+HN N
+N CA
+[ mapping ]
+CA BB
+C  BB
+O  BB
+OXT BB
+
+
+[modification]
+[ from ]
+universal
+[ to ]
+martini30dev
+[ from blocks ]
+N-ter
+[ to blocks ]
+N-ter
+[ from nodes ]
+C
+O
+[ from edges ]
+C O
+CA C
+[ mapping ]
+HN1 BB
+HN2 BB
+HN3 BB
+N   BB
+CA  BB
+C   BB
+O   BB
+
+[ modification ]
+[ from ]
+universal
+[ to ]
+martini30dev
```

Comment:
Am I seeing it correctly that here m3-dev, m3-beta mappings are in one file? Shouldn't they be distributed according to the same logic as the force-field directory?

Reply:
The mapping directory is a swamp that needs to be drained. But not now. We can decide on a sane folder structure once we take a look at the parsers.